### PR TITLE
fix(track): Send decisions for all experiments using an event when us…

### PR DESCRIPTION
**Pull request by rashidsp**
…ing track

### CODE REVIEW

File: lib/optimizely/event_builder.rb [hunk 1]
```diff
@@ -195,43 +195,38 @@ def get_conversion_params(event_key, event_tags, experiment_variation_map)
...
+      if event_tags
+        revenue_value = Helpers::EventTagUtils.get_revenue_value(event_tags, @logger)
+        event_object[:revenue] = revenue_value if revenue_value
 
-        single_snapshot[:events] = [event_object]
+        numeric_value = Helpers::EventTagUtils.get_numeric_value(event_tags, @logger)
+        event_object[:value] = numeric_value if numeric_value
 
-        conversion_event_params.push(single_snapshot)
+        event_object[:tags] = event_tags
```
Comment (msohailhussain) [inline lib/optimizely/event_builder.rb]: eventtags should have atleast one value then 'tags' should be added. Please add condition. 
https://github.com/optimizely/python-sdk/pull/136/files#diff-f5fd21234bea2bc5c0c2cbbf5931d93cR276

File: spec/spec_params.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@ module OptimizelySpec
     'version' => '2',
     'events' => [{
       'key' => 'test_event',
-      'experimentIds' => ['111127'],
+      'experimentIds' => %w[111127 122230],
```
Comment (msohailhussain) [inline spec/spec_params.rb]: why %w ?

Comment (rashidsp) [reply spec/spec_params.rb]: It's ruby convention for ['111127', '122230'], also rubocop suggestion.

**Comment by rashidsp** [comment]
Merged, closing it.

**Pull request closed without merging**